tests/common.go

```diff
@@ -219,6 +216,28 @@ func generateSupportBundle(ctx *scheduler.Context) {
 			expect(err).NotTo(haveOccurred())
 		})
 	})
+}*/
+
+// CollectSupport creates a support bundle
+func CollectSupport() {
+	context(fmt.Sprintf("generating support bundle..."), func() {
+		Step(fmt.Sprintf("save journal output on each node"), func() {
+			nodes, err := Inst().S.GetNodes()
+			expect(err).NotTo(haveOccurred())
+			expect(nodes).NotTo(beEmpty())
+
+			journalCmd := fmt.Sprintf(
+				"echo t > /proc/sysrq-trigger && journalctl -l > ~/all_journal_%v",
+				time.Now().Format(time.RFC3339))
+			for _, n := range nodes {
+				_, err := Inst().N.RunCommand(n, journalCmd, node.ConnectionOpts{
+					Timeout:         2 * time.Minute,
+					TimeBeforeRetry: 10 * time.Second,
+				})
+				expect(err).NotTo(haveOccurred())
```

Comment:
if this expect fails for one node will it try journalctl on the remaining nodes?